changelog/535.fixed.md

```diff
@@ -0,0 +1 @@
+Fix branch handling in `_run_transform` and `execute_graphql_query` functions.
```

Comment:
Would be good to update the message to indicate that it's only impacting `infrahubctl` not the SDK itself